docs/app/Examples/elements/List/Types/ListOrderedExample.js

```diff
@@ -0,0 +1,21 @@
+import React, {Component} from 'react';
+import {List, ListItem} from 'stardust';
+
+export default class ListOrderedExample extends Component {
+  render() {
+    return (
+      <List className='ordered'>
+        <ListItem description='Apples' />
+          <div>
+            <List>
+              <ListItem description='Fuji' />
+              <ListItem description='Granny Smith' />
+              <ListItem description='Honeycrisp' />
+            </List>
+          </div>
```

Comment:
Let's remove the extra div here.  Turns out there is an issue with the `ListItem` component.  It currently renders `children || description` as the content.  I'll open up a separate issue for the `ListItem` as there are number of fixes it needs.